euth/organisations/autofixtures.py

```diff
@@ -0,0 +1,14 @@
+from .models import Organisation
+from autofixture import register, AutoFixture
+from faker import Faker
+from .. import generators
+
+
+class OrganisationAutoFixture(AutoFixture):
+
+    field_values = {
+        'name': generators.FakeOrganisationNameGenerator(),
```

Comment:
It's not wrong to implement a `FakeOrganisationNameGenerator` class or other classes. But you can also use `autofixture.generators.CallableGenerator()` and call `faker` directly like so:

```python
from autofixture import AutoFixture, generators, register
from faker import Factory

from . import models

fake = Factory.create()


class OrganisationAutoFixture(AutoFixture):

    field_values = {
        'name': generators.CallableGenerator(fake.company),
        'slug': generators.CallableGenerator(fake.slug),
    }


register(models.Organisation, OrganisationAutoFixture)
```

This reduces the amount of code you have to maintain and that can break. 😃